site/public/js/server.js

```diff
@@ -133,33 +141,79 @@ function changeDiffView(div_name, gradeable_id, who_id, version, index, autochec
 
 }
 
+
+/**
+ * Collapses Test Case Output.  
+ *
+ * @param {string} div_name - id of div containing the output of the test case
+ * @param {string} index - index of test case
+ * @param {object} loadingTools - loading tools used by span containing the text that 
+ *                                correspond to the state of the test case (show, hide, loading)
+ */
+function CollapseTestcaseOutput(div_name, index, loadingTools)
+{
```

Comment:
The opening curly brace should be on the same line as the function declaration, as per the Submitty [JS style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript).  This file is not linted by `eslint`, but the same rules should be applied when possible.  The specific eslint rule we use is the `brace-style` [rule](https://eslint.org/docs/rules/brace-style).